lib/publiccloud/provider.pm

```diff
@@ -47,7 +47,7 @@ sub init {
         my $cloud_name = $self->conv_openqa_tf_name;
         assert_script_run('cd ' . TERRAFORM_DIR);
         assert_script_run('git clone --depth 1 --branch ' . get_var('HA_SAP_GIT_TAG', 'master') . ' ' . get_required_var('HA_SAP_GIT_REPO') . ' .');
-        assert_script_run('cp pillar_examples/automatic/{cluster.sls,hana.sls,top.sls} salt/hana_node/files/pillar/');
+        assert_script_run("cp pillar_examples/automatic/${_}/* salt/${_}_node/files/pillar/") foreach (qw(hana drbd));
```

Comment:
maybe better to read:
```suggestion
        assert_script_run('for i in hana drbd; do cp pillar_examples/automatic/${i}/* salt/${i}_node/files/pillar/';done);
```

Reply:
I would be against this change, as the result for the `assert_script_run` call will depend only on the return code of `cp pillar_examples/automatic/drbd/* salt/drbd_node/files/pillar/`, no matter whether `cp pillar_examples/automatic/hana/* salt/hana_node/files/pillar/` does work or fail.

Reply:
Me too.

@cfconrad but I can do a "regular" `foreach` call with a variable if you prefer. It was to save some lines, and I found it easily readable (and I'm not a Perl expert at all ;-)).

Reply:
just keep it :)